compiler/optimizer/Inliner.cpp

```diff
@@ -303,8 +303,13 @@ void TR_InlinerBase::setInlineThresholds(TR::ResolvedMethodSymbol *callerSymbol)
 
     _callerWeightLimit -= size;
 
-    _nodeCountThreshold
-        = (comp()->getOption(TR_NotCompileTimeSensitive) || comp()->getMethodHotness() >= hot) ? 16000 : 3000;
+    if (TR::Options::getAggressivityLevel() == TR::Options::TR_AggresivenessLevel::AGGRESSIVE_THROUGHPUT) {
+        _nodeCountThreshold = 8000;
```

Comment:
I think we should have an env var to take this new value back to the older one if we need to try something for some reported regression.

Isn't `TR_NotCompileTimeSensitive` set already under `-Xtune:throughput` ?

Reply:
> Are'nt we checking for -Xtune:throughput on the line https://github.com/eclipse-openj9/openj9/blob/0b13b6cdea94f3d8cfce1eaed80533cfad473067/runtime/compiler/control/CompilationThread.cpp#L9109 ?

Yeah you're right, I didn't see that part. That means that `comp()->getOption(TR_NotCompileTimeSensitive) || comp()->getMethodHotness() >= hot` will be true for all methods under `-Xtune:throughput`, so do we want to change 16000 to 8000 then?

Reply:
Note that "not compile time sensitive" can be true for other cases too, e.g. AOT compiles. I don't think we want to affect all those other cases too. So, maybe we need to check `-Xtune:throughput` after all.

Reply:
> Thanks, I will approve and start testing but one small suggestion to change `static` to `static const` ?

Done.